packages/ng-form/src/lib/components/form/form.component.html

```diff
@@ -0,0 +1,11 @@
+<form
+  [formGroup]="form"
+  [class]="formStyleClass"
+  (ngFormSubmit)="submitFormSave()">
+  <div [class]="contentStyleClass">
+    <ng-content select="[content]"></ng-content>
+  </div>
+  <div [class]="footerStyleClass" *ngIf="showFooter">
+    <ng-content select="[footer]"></ng-content>
+  </div>
```

Comment:
Not sure this logic should be here 🤔 I expected the `ng-form` library to just give the utilities to better work with form, not to implement the layout of my app.